### PR TITLE
HTML-Forms Project

### PR DESCRIPTION
**Pull request by Angelesh1995**
This project consists of building an HTML page and form that matches the INTUIT Sign Up page.

### CODE REVIEW

**Comment by kobiyoyo** [comment]
### Pull Request under review, by Adama Chubiyojo Desmond (TSE)

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team   
#### Great Job so far, but there are few more things required. 

## General
- [ ] Please let's add linters to your project.
- https://github.com/microverseinc/linters-config/tree/master/css

- [ ] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload and fix errors.


## Form
- [ ] The Phone `<input>` type shouldn't be `text` 

[OPTIONAL] -  [`<nav>` tag](http://html5doctor.com/nav-element/) should be used for  major navigation links

### Make changes and submit another code review request.
*Happy coding*

File: index.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                    target="_blank">Learn more</a>
+                </div>
+                <fieldset>
+                  <label for="email">Email Address
+                  </label>
+                  <input type="email" name="email-address" id="email" placeholder="" required>
+
+                  <label for="text">Phone (Recommended)
+                  </label>
+                  <input type="text" name="phone-number" id="text">
```
Comment (kobiyoyo) [inline index.html]: I recommend using **tel** for input type

**Review by kobiyoyo — CHANGES_REQUESTED** [review]

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — COMMENTED** [review]
## Status: Require changes 🔴 

Hi, @Angelesh1995  and @dansantander,:+1: great job implementing some of the changes requested by the previous reviewer. However, there are some requirements left out. Find comment below: 
- Kindly enable stickler this repo. Go [here](https://stickler-ci.com/) to enable stickler.
- Kindly fix the error returned by [css validator](https://jigsaw.w3.org/css-validator)
fsleep

Make these changes and submit code review 🙂 
Happy coding 💻 
Daniel Larbi Addo

**Pull request closed without merging**